benchexec/tools/coastal.py

```diff
@@ -0,0 +1,54 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+Copyright (C) 2007-2018  Dirk Beyer
+All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+    Tool info for COASTAL
+    (http://www.cs.sun.ac.za/coastal/).
+    """
+
+    REQUIRED_PATHS = ["coastal-sv-comp"]
```

Comment:
Is this really the only required file for executing your tool? If yes, you can remove this line, because the default implementation of `program_files` adds the executable anyway. If not, please add all other required directories and files.